### PR TITLE
okd: add optional and adhoc jobs for 4.11/4.12/4.13

### PR DESCRIPTION
**Pull request by vrutkovs**
Add OKD-based optional GCP job so that PRs can be checked on  OKD too

### CODE REVIEW

**Comment by LorbusChris** [comment]
/assign @sjenning
Could you take a look?

**Comment by LorbusChris** [comment]
ping @sjenning

**Comment by LorbusChris** [comment]
This PR has been open for 3 months now.
/assign @bparees
/assign @deads2k

File: ci-operator/config/openshift/origin/openshift-origin-release-4.12__okd.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+releases:
+  initial:
+    integration:
+      name: "4.11"
```
Comment (bparees) [inline ci-operator/config/openshift/origin/openshift-origin-release-4.12__okd.yaml:10]: 4.12?

File: ci-operator/config/openshift/origin/openshift-origin-release-4.12__okd.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      namespace: origin
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.11"
```
Comment (bparees) [inline ci-operator/config/openshift/origin/openshift-origin-release-4.12__okd.yaml:15]: 4.12?

File: ci-operator/config/openshift/origin/openshift-origin-release-4.10__okd.yaml [hunk 3]
```diff
@@ -7,12 +7,12 @@ build_root:
 releases:
   initial:
     integration:
-      name: "4.10"
+      name: "4.12"
```
Comment (bparees) [inline ci-operator/config/openshift/origin/openshift-origin-release-4.10__okd.yaml:10]: why is this changing to 4.12 in a file that contains 4.10 in the name?

File: ci-operator/config/openshift/origin/openshift-origin-release-4.10__okd.yaml [hunk 4]
```diff
@@ -7,12 +7,12 @@ build_root:
...
       namespace: origin
   latest:
     integration:
       include_built_images: true
-      name: "4.10"
+      name: "4.12"
```
Comment (bparees) [inline ci-operator/config/openshift/origin/openshift-origin-release-4.10__okd.yaml:15]: why is this changing to 4.12 in a file that contains 4.10 in the name?

Comment (vrutkovs) [reply ci-operator/config/openshift/origin/openshift-origin-release-4.10__okd.yaml]: Oops, must have copypasted it wrong, sorry

**Pull request merged**